commands/apps/index.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+let co  = require('co');
+let cli = require('heroku-cli-util');
+let _   = require('lodash');
+
+function* run (context, heroku) {
+  let org = (!context.flags.personal && !context.flags.all && context.org) ? context.org : null;
+  let space = context.flags.space;
+
+  function regionizeAppName (app) {
+    if (app.region && app.region.name !== 'us') {
+      return `${app.name} (${cli.color.green(app.region.name)})`;
+    } else {
+      return app.name;
+    }
+  }
+
+  let isOrgApp = app => !app.owner.email.endsWith('@herokumanager.com');
+
+  function listApps (apps) {
+    apps.forEach(app => cli.log(regionizeAppName(app)));
+    cli.log();
+  }
+
+  function print (apps, user) {
+    if (apps.length === 0) {
+      if (space)    cli.log(`${cli.color.green(space)} has no apps.`);
+      else if (org) cli.log(`${cli.color.magenta(org)} has no apps.`);
+      else          cli.log('You have no apps.');
+      return;
+    } else if (space) {
+      cli.styledHeader(`Apps in space ${cli.color.green(space)}`);
+      listApps(apps);
+    } else if (org) {
+      cli.styledHeader(`Apps in organization ${cli.color.magenta(org)}`);
+      listApps(apps);
+    } else {
+      apps = _.partition(apps, app => app.owner.email === user.email);
+      if (apps[0].length > 0) {
+        cli.styledHeader('My Apps');
+        listApps(apps[0]);
+      }
+
+      if (apps[1].length > 0) {
+        cli.styledHeader('Collaborated Apps');
+        cli.table(apps[1], {
+          printHeader: false,
+          columns: [
+            {key: 'name', get: regionizeAppName},
+            {key: 'owner.email'},
+          ]
+        });
+      }
+    }
+  }
+
+  let requests = yield {
+    apps: org ? heroku.get(`/organizations/${org}/apps`) : heroku.get('/apps'),
```

Comment:
@dickeyxxx The problem is that in the plugin we do not translate the space to an org.
```
-> % heroku apps --space rbriggs-space
--> GET /apps
```
So this would only get org apps in the space that I am a collaborator on, rather than all org apps within that space.  It should be doing a lookup to `https://api.heroku.com/spaces/{space}` then grabbing apps from `https://api.heroku.com/organizations/{body['organization']['name']}/apps`